app/src/main/java/com/zeapo/pwdstore/utils/Result.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+
+package com.zeapo.pwdstore.utils
+
+/**
+ * Emulates the Rust Result enum but without returning a value in the [Ok] case.
+ * https://doc.rust-lang.org/std/result/enum.Result.html
+ */
+sealed class Result {
+
+    object Ok : Result()
```

Comment:
Add a generic type here so it can also be used when we want to pass the result value back. Also, I would prefer the naming to be Success and Error

Reply:
It's gonna complicate the current use case for no reason. We'll do this when we need it.